modules/microshift-low-latency-install-kernelrt-rpm.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * microshift_configuring/microshift-low-latency.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-low-latency-install-kernelrt_{context}"]
+= Installing the {op-system-rt-kernel}
+
+While the real-time kernel is not necessary for low latency workloads, using the {op-system-rtk} can optimize low latency performance. You can install it on a host using RPM packages, and include it in a {op-system-ostree-first} image deployment.
+
+.Prerequisites
+* You have a Red Hat subscription that includes {op-system-rt-kernel}. For example, your host machine is registered and Red Hat Enterprise Linux (RHEL) is attached to a RHEL for Real Time subscription.
+* You are using x86_64 architecture.
+
+.Procedure
+. Enable the {op-system-rtk} repository by running the following command:
++
+[source,terminal]
+----
+$ sudo subscription-manager repos --enable rhel-9-for-x86_64-rt-rpms
+----
+
+. Install the real-time kernel by running the following command:
++
+[source,terminal]
+----
+$ sudo dnf install kernel-rt -y
```

Comment:
Might want to keep it as `dnf install -y` as above.

Reply:
agree